smact/tests/test_core.py

```diff
@@ -87,6 +91,12 @@ def test_harrison_gap_MgCl(self):
             3.545075110572662,
         )
 
+    def test_valence_electron_count(self):
+        self.assertAlmostEqual(valence_electron_count("Fe2O3"), 6.8, places=2)
+        self.assertAlmostEqual(valence_electron_count("CuZn"), 11.5, places=2)
+        with self.assertRaises(ValueError):
+            valence_electron_count("Xx2O3")  # Xx is not a real element

```

Comment:
This is causing the CI to fail because it doesn't catch the `NameError` due to Xx not being real, which causes the test to fail. Make that you're capturing the correct error to get this to pass.